UOP1_Project/Assets/Scripts/Factory/DOCUMENTATION.md

```diff
@@ -0,0 +1,112 @@
+# Factory
+
+![Factory Diagram](https://i.imgur.com/8cleBXk.png)
```

Comment:
Is there a reason for hosting this on imgur other than convenience? It seems like they're usually stored alongside the documentation contents (e.g. https://github.com/UnityTechnologies/ScriptableRenderPipeline/tree/master/com.unity.render-pipelines.lightweight/Documentation~/images)

Reply:
Just following the precedent set on this repo. The [main README](https://raw.githubusercontent.com/UnityTechnologies/open-project-1/main/README.md) images are all hosted on imgur. But I can re-export diagrams and put them wherever.

Reply:
Ah, didn't look at existing images in this repo!